app/com/linkedin/drelephant/mapreduce/TaskLevelAggregatedMetrics.java

```diff
@@ -106,6 +106,9 @@ private void compute(MapReduceTaskData[] taskDatas, long containerSize, long ide
     }
 
     for (MapReduceTaskData taskData: taskDatas) {
+      if (!taskData.isTimeAndCounterDataPresent()) {
```

Comment:
Can you add a comment to the function saying that the aggregatemetrics are not expected to be accurate when sampling is enabled so that it's clear ?